src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/_catalog.yml

```diff
@@ -27,13 +27,7 @@ _features:
   prefix: /features
 
 _attributes:
-  resource: "attribute.yml"
```

Comment:
As discussed with @jolelievre, we decided to merge `attribute.yml` and `attribute_group.yml`. Like that, we can handle routes with `{attributeGroupId}` parameter and those without it in the same file.